### PR TITLE
feat(elements): add support for Vue3

### LINKED ISSUE #200: Migrate elements-example-vue from Vue v2 to v3
### Describe the bug
We have a [vue example project](https://github.com/inovex/elements-example-vue/pull/3) which is currently based on Vue 2.x. Now that Vue 3 got released, we should also update our example project AND our migration guide to support both versions. 

I made some first steps here: https://github.com/inovex/elements-example-vue/pull/3

But there are still some issues to address:
- Everything below the `ino-button` won't be displayed (maybe the way the attributes of the other components are used is deprecated?)
- I had to use the beta version of the `vue-loader` in order to compile successfully (see https://github.com/vuejs/vue-next/issues/1414#issuecomment-648998826). Is there any other way to make it work?
- Upgrade our integration guide to support both versions (in this repo)

### PR DESCRIPTION
**Pull request by BenPag**
Relates to #467
Closes #200 

## Proposed Changes

- add example vue project to monorepo
- migrate example project to Vue.js 3
- use latest elements version
- use new wrapper package `elements-vue`

## Things to check

- [ ] Does the change need to be documented?
- [ ] Does any existing example code needs to be updated?
- [ ] Is the change properly tested?
- [ ] Is it helpful to provide another example to demonstrate the new feature?
- [ ] Are there other code lines that need to be modified?

### CODE REVIEW

**Comment by BenPag** [comment]
is blocked by #552

**Comment by janivo** [comment]
Maybe we should rename `elements-example-vue` to `elements-vue-example` so it will be sorted just below the vue-wrapper

File: packages/elements-example-vue/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
```
Comment (pfecht) [inline packages/elements-example-vue/package.json:1]: BTW: Why don't we use vue with typescript? Is this not recommended? If recommended, let's create a follow up story for that.

File: packages/elements-example-vue/src/main.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { createApp } from 'vue';
+import { addIcons } from '@inovex.de/elements/dist/collection/util/icons';
+import { ICON_PATHS } from '@inovex.de/elements/dist/inovex-elements/ino-icon/index.esm.js';
+import App from './App.vue'
+
+addIcons(ICON_PATHS); // c) (only needed if you want to use our icons)
+
+const app = createApp(App);
```
Comment (pfecht) [inline packages/elements-example-vue/src/main.js:8]: As mentioned, would be nicer to just use `InoElementsVue` or so and call `.use` here :)

File: packages/storybook/src/stories/docs/framework-integration/vue-instructions.stories.mdx [hunk 3]
```diff
@@ -3,20 +3,74 @@ import { Meta } from "@storybook/addon-docs";
...
+$ npm install @inovex.de/elements-vue --save
+```
+
+or **yarn**:
+
+```sh
+$ yarn add @inovex.de/elements-vue
+```
+
+### 2) Prepare the Project to use icons (optional)
```
Comment (pfecht) [inline packages/storybook/src/stories/docs/framework-integration/vue-instructions.stories.mdx:32]: Also Update Documentation here.

**Review by pfecht — CHANGES_REQUESTED** [review]
@BenPag

File: packages/elements-example-vue/package.json (hunk 1 above)
Comment (BenPag) [reply packages/elements-example-vue/package.json:1]: I will create a follow up story :)

File: packages/elements-vue/package.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@inovex.de/elements-vue",
+  "version": "6.0.0",
```
Comment (janivo) [inline packages/elements-vue/package.json:3]: This version should match the current master

File: packages/elements-vue/package.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    "dist/"
+  ],
+  "scripts": {
+    "build": "npm run clean && npm run compile",
+    "clean": "shx rm -rf dist",
+    "compile": "npm run tsc",
+    "tsc": "tsc -p ."
+  },
+  "dependencies": {
+    "@inovex.de/elements": "6.0.0"
```
Comment (janivo) [inline packages/elements-vue/package.json:29]: Bump to current version

File: packages/elements-vue/package.json [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  },
+  "devDependencies": {
+    "@stencil/core": "^2.14.0",
```
Comment (janivo) [inline packages/elements-vue/package.json:32]: Do we really need stencil as a dependency?

File: packages/storybook/src/stories/docs/framework-integration/vue-instructions.stories.mdx [hunk 7]
```diff
@@ -3,20 +3,77 @@ import { Meta } from "@storybook/addon-docs";
 <Meta title="Docs/Framework Integration/Vue" />
 
 # Integrate with Vue.js (as Web Components)
-
-> We set up a demo project with Vue.js to show you how to integrate the components into a newly generated Vue.js
+> We set up a demo project with Vue.js (v3) to show you how to integrate the components into a newly generated Vue.js
 > project. Scroll down to see the Stackblitz.
 >
 > These instructions are based on `vue@2.6.11` and [`create-react-app@4.2.3`](https://cli.vuejs.org/).
```
Comment (janivo) [inline packages/storybook/src/stories/docs/framework-integration/vue-instructions.stories.mdx:9]: We use a newer vue version, don't we?

File: packages/storybook/src/stories/docs/framework-integration/vue-instructions.stories.mdx [hunk 8]
```diff
@@ -3,20 +3,77 @@ import { Meta } from "@storybook/addon-docs";
...
+Open `src/main.js` and add the following code snippet on the first lines to use the elements vue wrapper.
+
+```js
+// src/main.js
+import { createApp } from 'vue';
+import { InoElementsVue } from '@inovex.de/elements-vue';
+import App from './App.vue'
+
+const app = createApp(App);
+app.use(InoElementsVue);
```
Comment (janivo) [inline packages/storybook/src/stories/docs/framework-integration/vue-instructions.stories.mdx:43]: Maybe we should provide a comment to this line to make it clear that this is the line the consumer has to add

e.g.: `app.use(InoElementsVue); // add this line to your code`

**Review by janivo — CHANGES_REQUESTED** [review]
Really nice work overall!
One thing I mentioned before: I think it would be nice to rename the `elements-example-vue` to `elements-vue-example` so it will be sorted just below the vue-wrapper

File: packages/storybook/src/stories/docs/framework-integration/vue-instructions.stories.mdx (hunk 7 above)
Comment (BenPag) [reply packages/storybook/src/stories/docs/framework-integration/vue-instructions.stories.mdx:9]: I removed the specific lines which contains version numbers.

**Comment by BenPag** [comment]
Do we add `elements-vue-example` to `commit-scope-enum.js`?

**Comment by janivo** [comment]
> Do we add `elements-vue-example` to `commit-scope-enum.js`?

Yes but without the components (e.g. no `elements-vue-example|ino-button`)

**Review by janivo — APPROVED** [review]

**Review by janivo — APPROVED** [review]

**Pull request merged**